pkg/skaffold/docker/reference_test.go

```diff
@@ -72,6 +79,7 @@ func TestParseReference(t *testing.T) {
 			parsed, err := ParseReference(test.image)
 
 			testutil.CheckErrorAndDeepEqual(t, false, err, test.expectedName, parsed.BaseName)
+			testutil.CheckErrorAndDeepEqual(t, false, err, test.expectedTag, parsed.Tag)
```

Comment:
Two instances of `CheckErrorAndDeepEqual` could be replaced with `CheckDeepEqual` here